apps/examples/testcase/le_tc/kthread_stack_protection/kthread_stack_protection_tc_main.c

```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file kthread_stack_protection_tc_main.c
+
+/// @brief Main Function for Kthread Stack Protection TestCase Example
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <tinyara/testcase_drv.h>
+#include "tc_common.h"
+#include "tc_internal.h"
+
+static int g_tc_fd;
+
+int tc3_get_drvfd(void)
```

Comment:
Please add static for this function.

Reply:
Modified design for TC. This part of code is removed now.